module #2. REST API Basics/rest-api-basic/core/src/main/java/com/epam/esm/exception/AbstractException.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.esm.exception;
+
+
+public abstract class AbstractException extends RuntimeException {
```

Comment:
AbstractException is redundant. It makes more complexity and didn't have any value. This abstraction layer is useless. It can be removed and child exception classes can be extended from RuntimeException